vcenter/vcenter.py

```diff
@@ -391,9 +391,17 @@ def get_machine_info(self, uuid):
 
     def wait_for_task(self, task):
         while (task.info.state == 'running' or task.info.state == 'queued'):
+            message = "no-message"
+            progress = ""
+            try:
+                progress = task.info.progress
+                message = task.info.description.message
+            except Exception:
+                pass
```

Comment:
You shall not `pass`!

The preferred way would be to place current logging line 402 right before `except Exception`, and in case of exception log at least that the "attempt to retrieve the data failed". This way we will be able to track how often does it happen.